app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,50 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+
+    category = benchmark_run.result.keys[0]
```

Comment:
Yes that is correct.

On Jul 25, 2016 7:22 PM, "Saurabh Shah" <notifications@github.com> wrote:

> In app/jobs/benchmark_regression_job.rb
> <https://github.com/ruby-bench/ruby-bench-web/pull/174#discussion_r72047206>
> :
>
> > @@ -0,0 +1,50 @@
> > +require 'uri'
> > +require 'net/http'
> > +require 'json'
> > +
> > +class BenchmarkRegressionJob < ActiveJob::Base
> > +  queue_as :default
> > +
> > +  def perform(benchmark_run_id)
> > +
> > +    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
> > +
> > +    category = benchmark_run.result.keys[0]
>
> So let's say if there are two categories. Then do we need to check for
> both the categories and create an issue if any one of them fails? Basically
> checking in a loop for all the categories.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ruby-bench/ruby-bench-web/pull/174/files/7d68a3221005f8e2781ddd304ea528175c11f072#r72047206>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEIofrAs4NzcYr5Z9viIWnW__NH5t73Rks5qZJxTgaJpZM4JFDxj>
> .
>